docs/sources/source/asr/tutorial.rst

```diff
@@ -2,19 +2,19 @@ Tutorial
 ========
 
 Make sure you have installed ``nemo`` and ``nemo_asr`` collection.
-See :ref:`installation` section.
+See the :ref:`installation` section.
 
 .. note::
-    You only need `nemo` and `nemo_asr` collection for this tutorial.
+    You only need the `nemo` and `nemo_asr` collections for this tutorial.
```

Comment:
`nemo_asr` is a collection, `nemo` isn't.

Reply:
we should also keep line 4 and 8 consistent

Reply:
Let's make this line `installed ``nemo`` and the ``nemo_asr`` collection.`
And make line 8 similar.